plugins/@grouparoo/zendesk/src/lib/export/exportProfile.ts

```diff
@@ -37,10 +42,14 @@ export const exportProfile: ExportProfilePluginMethod = async ({
       oldProfileProperties
     );
 
+    // tags need to be formatted
+    const newTags = newGroups.map((groupName) => makeTagName(groupName));
+    const oldTags = oldGroups.map((groupName) => makeTagName(groupName));
+
     if (toDelete) {
       if (user) {
         // this does a soft delete
-        await client.users.delete(user.id);
+        await deleteContact(client, syncOperations, user, true);
```

Comment:
Was this added because we were doing something more complex here before? I believe we can just check the syncOp before calling the client function like you're doing in the update/create cases.